pkg/operator2/operator.go

```diff
@@ -521,7 +530,12 @@ func (c *authOperator) checkDeploymentReady(deployment *appsv1.Deployment, opera
 func (c *authOperator) checkRouteHealthy(route *routev1.Route, routerSecret *corev1.Secret, ingress *configv1.Ingress) (ready bool, msg, reason string, err error) {
 	caData := routerSecretToCA(route, routerSecret, ingress)
 
-	rt, err := transportFor("", caData, nil, nil)
+	// if systemCABundle is not empty, append the new line to the caData
+	if len(c.systemCABundle) > 0 {
+		caData = append(bytes.TrimSpace(caData), []byte("\n")...)
```

Comment:
nit: no need to trim the spaces here, there can be multiple empty lines between the certs
https://goplay.space/#2tCqSuOBWJL